cdc/entry/mounter.go

```diff
@@ -31,11 +31,12 @@ import (
 	"github.com/pingcap/tidb/table"
 	"github.com/pingcap/tidb/tablecodec"
 	"github.com/pingcap/tidb/types"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+
```

Comment:
```suggestion
```

Reply:
I have configurated my IDE to separate 3rd-party imports with packages from itself. Should I turn if off?